content/docs/guides/gitops-for-kubeflow.md

```diff
@@ -0,0 +1,147 @@
+
+## Argo CD and GitOps
+Argo CD is a Kubernetes-native Declarative Continuous Delivery tool that follows the GitOps methodology.  Along with all the benefits of using GitOps, Argo CD offers:
+
+* Integrations with templating tools like Ksonnet, Helm, and Kustomize in addition to plain yaml files to define the desired state of an application
+* Automated or manual syncing of applications to its desired state
+* Intuititve UI to provide observability into the state of applications
+* Extensive CLI to integrate Argo CD with any CI system
+* Enterprise-ready features like auditability, compliance, security, RBAC, and SSO
+
+## Before you start
+This guide assumes you have a kubernetes cluster. If you don't, follow [this guide](/docs/started/getting-started/#set-up-kubernetes) to create one.
+
+## Install Argo CD
+Follow the [argo cd getting started guide](https://github.com/argoproj/argo-cd/blob/master/docs/getting_started.md) up to the '[Create an application from a git repository location](https://github.com/argoproj/argo-cd/blob/master/docs/getting_started.md#6-create-an-application-from-a-git-repository-location)' step.
+
+* For example, if your local machine is OSX and you want to deploy Argo CD and Kubeflow to the same GKE cluster, you could run:
+
+1. Install Argo CD to the Kubernetes Cluster
+```shell
+ARGO_CD_LATEST=$(curl --silent "https://api.github.com/repos/argoproj/argo-cd/releases/latest" | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/')
+kubectl create namespace argocd
+kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/$ARGO_CD_LATEST/manifests/install.yaml
+```
+
+1. Install Argo CD Cli
+```shell
+brew install argoproj/tap/argocd
+```
+
+1. Set extra permissions for Argo CD
+Since the GKE cluster has RBAC enabled, you will need to grant your account the ability to create new cluster roles by running:
+```shell
+kubectl create clusterrolebinding YOURNAME-cluster-admin-binding --clusterrole=cluster-admin --user=YOUREMAIL@gmail.com
+```
+
+1. Expose the Argo CD API server
```

Comment:
Is it possible to connect via kubectl port-forward?
We generally try to avoid telling people to expose services via LoadBalancer because we don't want to assume they understand the security implications of doing so.

Reply:
That's a great point! I just changed the directions to take that into account.